WORKSPACE

```diff
@@ -2,11 +2,6 @@ workspace(name = "rules_jvm_external")
 
 android_sdk_repository(name = "androidsdk")
 
-local_repository(
```

Comment:
was this unused?

Reply:
Yes, unused

Reply:
Buildifier says this name was defined earlier by the workspace() declaration